en/docs/guides/login/oidc-parameters-in-auth-request.md

```diff
@@ -8,21 +8,40 @@ This page guides you through some special request parameters used with [OpenID C
 
 Use the [state parameter](../../../references/concepts/authentication/traditional-authentication-request#state-parameter) to maintain a correlation between the request and the response.
 
+```tab="Request Format"
+https://<IS_HOST>:<IS_PORT>/oauth2/authorize?
+response_type=<response_type>
+&client_id=<client_id>
+&state=<state_value>
+&redirect_uri=<callback_url>
+```
+
 ```tab="Sample Request"
-https://<host>:/authorize?
+https://localhost:9443/oauth2/authorize?
 response_type=token
 &client_id=s6BhdRkqt3
 &state=xyz
-&redirect_uri=<callback_url>
+&redirect_uri=https://localhost.com:8080/callback
+```
+
+You will receive a response similar to shown below.
+
+```tab="Response Format"
+HTTP/1.1 302 Found
+Location: <callback_url>#access_token=<access_token>
```

Comment:
Do you think we should use `https://<IS_HOST>:<IS_PORT>/callback#access_token=<access_token>` instead?

Reply:
this callback url is different. It is not the IS url